src/images/imagesCollection.js

```diff
@@ -1,12 +1,14 @@
 const roomImages = [
-    "https://user-images.githubusercontent.com/102932448/183271129-975542c9-868e-4a0b-bb38-32e2fe600ee4.jpeg",
-    "https://user-images.githubusercontent.com/102932448/183271151-d1175ef4-b568-4122-9272-b9624cdf0e3b.jpeg",
-    "https://user-images.githubusercontent.com/102932448/183271165-fe116bbf-673f-4b0f-9575-98f9508032ff.jpeg",
```

Comment:
making the rooms objects with a type that can match their roomType and a picture key containing the link